### PR TITLE
Normalise metric names from common standard

### PR DESCRIPTION
**Pull request by hauleth**
This allows users to define metrics in exporter independent way and then
let exporter handle the unification of the metric name to the exporter
requested format.

### CODE REVIEW

**Comment by deadtrickster** [comment]
where are the tests? ;-)

File: src/opencensus_influxdb.erl [hunk 1]
```diff
@@ -28,7 +28,7 @@ build_tags(Tags) ->
     lists:join($,, List).
 
 normalise(Atom) when is_atom(Atom) ->
-    normalise(erlang:atom_to_binary(Atom, utf8), <<>>);
+    normalise(erlang:atom_to_list(Atom, utf8), <<>>);
```
Comment (tsloughter) [inline src/opencensus_influxdb.erl]: Why this change?

Comment (ferd) [reply src/opencensus_influxdb.erl]: same question, it seems all normalization is done on binaries anyway. Might as well keep the binary as it is

File: src/opencensus_influxdb.erl [hunk 2]
```diff
@@ -38,6 +38,7 @@ normalise(<<",", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\,
 normalise(<<" ", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\ ">>);
 normalise(<<"=", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\=">>);
 normalise(<<"\"", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "\\\"">>);
+normalise(<<"/", Rest/binary>>, Result) -> normalise(Rest, <<Result/binary, "_">>);
```
Comment (ferd) [inline src/opencensus_influxdb.erl]: out of curiosity, why replace by an underscore rather than escaping like with all other special characters?

Comment (hauleth) [reply src/opencensus_influxdb.erl]: OpenCensus use `/` as a general purpose separator in metrics and InfluxDB for that purpose use `_`. That was the main reasoning behind using `_` there.

File: src/opencensus_influxdb.erl (hunk 1 above)
Comment (hauleth) [reply src/opencensus_influxdb.erl]: TBH I do not remember. Probably because first version used list to normalise instead of binary.

File: src/opencensus_influxdb.erl (hunk 2 above)
Comment (ferd) [reply src/opencensus_influxdb.erl]: understood. Might make sense to put it in a comment
